BackofficeBundle/EventSubscriber/DataTransformer/ValueTransformerManager.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace OpenOrchestra\BackofficeBundle\EventSubscriber\DataTransformer;
+
+/**
+ * Class ValueTransformerManager
+ */
+class ValueTransformerManager
+{
+    protected $transformers = array();
+
+    /**
+     * @param ValueTransformerInterface $valueTransformer
+     */
+    public function addStrategy(ValueTransformerInterface $valueTransformer)
+    {
+        $this->transformers[$valueTransformer->getName()] = $valueTransformer;
+    }
+
+    /**
+     * @param string $fieldType
+     * @param mixed  $value
+     *
+     * @return ValueTransformerInterface|null $valueTransformer
+     */
+    public function get($fieldType, $value)
```

Comment:
why do you name your method `get` in the manager and `transform` in the strategies?